### DISCUSSION TITLE
Add TeachingTip Control

### DISCUSSION THREAD

**Discussion by laolarou726** [AvaloniaUI/Avalonia#14386] [Ideas]
## Is your feature request related to a problem? Please describe.

Maybe provide an implementation of the TeachingTip control just like UWP

[MSDN (https://learn.microsoft.com/en-us/windows/apps/design/controls/dialogs-and-flyouts/teaching-tip)](https://learn.microsoft.com/en-us/windows/apps/design/controls/dialogs-and-flyouts/teaching-tip)

![image](https://github.com/AvaloniaUI/Avalonia/assets/25716486/9fedbb42-67f1-4f88-8235-1addda5b9315)

**Comment by timunie** [comment]
FluentAvalonia has

**Reply by laolarou726** [reply]
Thx!

**Comment by timunie** [comment]
This is a very Specialized control so I am not sure we want to have it in the main repo. I can assume that this one can go into Avalonia.Labs in the first place or be something of the Pro-Package. I will move this convertion to the idea 💡 section as this is where it should be discussed first.
